OpenXmlPowerTools/DocumentAssembler.cs

```diff
@@ -717,10 +717,18 @@ static object ContentReplacementTransform(XNode node, XElement data, TemplateErr
                                         XElement paragraph = tc.Elements(W.p).FirstOrDefault();
                                         XElement cellRun = paragraph.Elements(W.r).FirstOrDefault();
                                         string xPath = paragraph.Value;
+                                        bool cellIsOptional = false;
```

Comment:
I believe this could be simplified to:

```
    var xPath = paragraph.Value;
    var optionalString = "";
    try
    {
        var xElement = XElement.Parse(xPath);
        xPath = (string)xElement.Attribute(PA.Select);
        optionalString = (string)xElement.Attribute(PA.Optional);
    }
    catch { }

    try
    {
        newValue = EvaluateXPathToString(d, xPath, optionalString?.ToLower() == "true");
    }
```